app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,7 @@
+.user_avatar {
+  display: inline-block!important;
```

Comment:
try to avoid this `!important` thing..
better get to understand the styling priority and set style in a correct priority chain